pki/ssh/keys.go

```diff
@@ -49,6 +49,22 @@ func ED25519() (crypto.PrivateKey, error) {
 	return pk, err
 }
 
+func MarshalPrivateKey(privateKey crypto.PrivateKey) ([]byte, error) {
```

Comment:
Public methods should be commented - but is this even used outside the package. Does it need to be exported.

Reply:
I can unexport this but since the Unmarshal method would need to be exported, it made sense to make this exported too.